src/types.js

```diff
@@ -53,7 +53,7 @@ type CombinedQueryConjuction = 'INTERSECT' | 'UNION' | 'EXCEPT';
 
 export type CombinedQuery = {
   type: string, // pattern ^combined$
-  conjuction: CombinedQueryConjuction,
+  conjunction: CombinedQueryConjuction,
```

Comment:
...same error on the right-hand side as well 😄 
`CombinedQueryConjuction` -> `CombinedQueryConjunction`